.github/workflows/rubyonrails.yml

```diff
@@ -27,7 +26,7 @@ jobs:
           --health-retries 5
     env:
       RAILS_ENV: test
-      DATABASE_URL: "postgres://rails:password@localhost:5432/hostedgpt_test"
+      DATABASE_URL: "postgres://rails:password@localhost:5432/hostedgpt"
```

Comment:
What motivated the change of database name? I would have thought that using `hostedgpt` for the test environment would collide with the other environments?

Reply:
Ohhh, I see your regexing within database.yml. Got it.